deeppavlov/core/models/tf_model.py

```diff
@@ -167,87 +162,7 @@ def print_number_of_parameters():
         log.info('Total number of parameters equal {}'.format(total_num_parameters))
 
     def destroy(self):
-        for k in list(self.sess.graph.get_all_collection_keys()):
-            self.sess.graph.clear_collection(k)
-
-
-class LRScheduledTFModel(TFModel, LRScheduledModel):
```

Comment:
Why delete LRScheduledTFModel?!

Reply:
sorry. I merged with dev and lost this class